install.sh

```diff
@@ -148,6 +163,9 @@ if ! [[ $silent ]] && ! [[ $no_modify_config ]]; then
       ;;
     esac
   done
+elif [[ $silent ]] && [[ $append_to_config ]]; then
```

Comment:
you can remove the `while ! [ $silent ]` check in line `150` in favor of `while ! [ $append_to_config ]`, and remove `[[ silent ]]` check here. This will cause interactive mode to also automatically append if the option is specified